libcorn/src/lua.rs

```diff
@@ -0,0 +1,51 @@
+use mlua::prelude::*;
+use crate::Value;
+use mlua::Value as OtherValue;
+impl<'lua> ToLua<'lua> for Value<'lua> {
+    fn to_lua(self, lua: &'lua Lua) -> LuaResult<LuaValue<'lua>> {
+        match self {
+            Value::Object(map) => {
+                let table = lua.create_table()?;
+                for (k, v) in map {
+                    table.set(k, v)?;
+                }
+                Ok(mlua::Value::Table(table))
+            }
+            Value::Array(vec) => {
+                let table = lua.create_table()?;
+                for (i, v) in vec.into_iter().enumerate() {
+                    table.set(i + 1, v)?;
+                }
+                Ok(mlua::Value::Table(table))
+            }
+            Value::String(s) => Ok(s.to_lua(lua)?),
+            Value::Integer(i) => Ok(i.to_lua(lua)?),
+            Value::Float(f) => Ok(f.to_lua(lua)?),
+            Value::Boolean(b) => Ok(b.to_lua(lua)?),
+            Value::Null(_) => Ok(LuaValue::Nil),
+        }
+    }
+}
+
+fn lua_parse(lua: &Lua, config: String) -> LuaResult<LuaTable> {
+    let globals = lua.globals();
+    let res = crate::parse(&config);
+    let table = lua.create_table()?;
+    match res {
+        Ok(v) => {
+            let val = mlua::ToLua::to_lua(v, lua)?;
+            table.set("value", val)?;
+        }
+        Err(e) => {
+            table.set("error", e.to_string())?;
+        }
+    }
+    Ok(table)
```

Comment:
`mlua` will [already handle panics](https://github.com/khvzak/mlua#panic-handling). so I'd be surprised if it doesn't handle errors for you (esp since the func returns a `Result`). Could you return a `LuaResult<LuaValue>` instead, removing the table, and have it handle the error for you?